content/docs/configuration.md

```diff
@@ -58,6 +58,9 @@ and config file path are set every time by default).
 #### upstream_ref
 
 (*string*) Git reference to last upstream git commit (for source-git repos).
+Can be set to *fixed* commit hash, tag or a branch. You can also use globbing pattern
```

Comment:
All commit hashes are *fixed*, aren't they? :smiley: 

```suggestion
Can be set to commit hash, tag or a branch name. You can also use globbing pattern
```

Reply:
Seemed like a good word ordering 🤔 anyways, it was meant to emphasize fixed branch (like `stable`) or tag... maybe replace with `specific`? or remove like you suggested